app/v1/controllers/AmendLossClaimController.scala

```diff
@@ -82,4 +92,38 @@ class AmendLossClaimController @Inject()(val authService: EnrolmentsAuthService,
       case DownstreamError => InternalServerError(Json.toJson(errorWrapper))
     }
   }
+
+
+  private def createAuditDetails(nino: String,
```

Comment:
Can this be replaced by the class of the same name?

Reply:
It could be in the companion object for AmendLossClaimAuditDetail maybe. I'll see how it looks.

Reply:
Have also added method to AuditResponse companion to reflect fact that either have errors or response body (removing ambiguity) which seems better, and can be re-used across controllers.